e2e/cypress/integration/admin/domain.js

```diff
@@ -23,8 +19,7 @@ describe("domain landing page", () => {
     // eslint-disable-next-line cypress/no-unnecessary-waiting
     cy.wait(500)
     // check not in one string because it can be different order
-    cy.contains("SAP Converged Cloud")
-    cy.contains("button", "Log in")
+    cy.contains("button", "Enter CC3TEST")
```

Comment:
Would be better hier to check for an fix button.id instead with the button text? If we change the button text the test would then fail...

Reply:
this is on purpose to check that the automatic generation of the button title is working, the test is running on `CC3TEST` domain so we should see the button with this text

Reply:
I was meaning generally in the tests. But in this specific one I would do following:

```js
cy.get("#yourButtonID").should("have.text", "Enter CC3TEST");
```